 Wed, 04 Mar 2020 13:14:38  INFO logger.py <module> [ line:20 ] hehehehe
 Wed, 04 Mar 2020 13:14:38  ERROR logger.py <module> [ line:21 ] heheheheheheh
Traceback (most recent call last):
  File "D:\Program Files (x86)\python\lib\site-packages\_pytest\config\__init__.py", line 456, in _importconftest
    return self._conftestpath2mod[key]
KeyError: WindowsPath('D:/python/mystudy/AppFramework/TestCases/conftest.py')
 Wed, 04 Mar 2020 13:14:38  INFO conftest.py init_session [ line:65 ] xxxxxxx
 Wed, 04 Mar 2020 13:14:52  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/btn_login')查找成功，耗时0.5377745628356934s
 Wed, 04 Mar 2020 13:14:52  INFO test_0_login.py test_prepare [ line:19 ] app已登录,执行数据清空操作!
 Wed, 04 Mar 2020 13:15:39  INFO test_0_login.py test_login_tele_wrong [ line:27 ] 手机号输入错误用例
 Wed, 04 Mar 2020 13:15:39  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Wed, 04 Mar 2020 13:15:40  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.561337947845459s
 Wed, 04 Mar 2020 13:15:40  INFO basepage.py get_ele [ line:47 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Wed, 04 Mar 2020 13:15:41  INFO basepage.py click_ele [ line:80 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Wed, 04 Mar 2020 13:15:42  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时0.4826540946960449s
 Wed, 04 Mar 2020 13:15:42  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Wed, 04 Mar 2020 13:15:43  INFO basepage.py input_ele_text [ line:59 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入 
 Wed, 04 Mar 2020 13:15:43  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Wed, 04 Mar 2020 13:15:43  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Wed, 04 Mar 2020 13:15:45  INFO basepage.py imgs_screenshot [ line:105 ] 截图成功，文件为D:\python\mystudy\AppFramework\Outputs\Screen_Shots\手机号输入错误-2020-03-04-13-15-44.png
 Wed, 04 Mar 2020 13:15:45  ERROR basepage.py get_ele [ line:51 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找失败！
Traceback (most recent call last):
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 46, in get_ele
    ele = self.driver.find_element(*loc)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\webdriver.py", line 282, in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.

 Wed, 04 Mar 2020 13:15:45  INFO test_0_login.py test_login_tele_wrong [ line:35 ] Test Fail
 Wed, 04 Mar 2020 13:15:45  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Wed, 04 Mar 2020 13:15:45  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Wed, 04 Mar 2020 13:15:59  WARNING connectionpool.py urlopen [ line:744 ] Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /wd/hub/session/d7ed8ea7-2799-469b-910e-60b0008c905a/appium/device/current_activity
 Wed, 04 Mar 2020 13:16:34  INFO test_0_login.py test_login_tele_wrong [ line:27 ] 手机号输入错误用例
 Wed, 04 Mar 2020 13:16:34  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Wed, 04 Mar 2020 13:16:35  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.5775768756866455s
 Wed, 04 Mar 2020 13:16:35  INFO basepage.py get_ele [ line:47 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Wed, 04 Mar 2020 13:16:36  INFO basepage.py click_ele [ line:80 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Wed, 04 Mar 2020 13:16:37  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.075204849243164s
 Wed, 04 Mar 2020 13:16:37  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Wed, 04 Mar 2020 13:16:38  INFO basepage.py input_ele_text [ line:59 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入15264
 Wed, 04 Mar 2020 13:16:38  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Wed, 04 Mar 2020 13:16:38  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Wed, 04 Mar 2020 13:16:39  INFO basepage.py imgs_screenshot [ line:105 ] 截图成功，文件为D:\python\mystudy\AppFramework\Outputs\Screen_Shots\手机号输入错误-2020-03-04-13-16-39.png
 Wed, 04 Mar 2020 13:16:39  ERROR basepage.py get_ele [ line:51 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找失败！
Traceback (most recent call last):
  File "d:\python\mystudy\AppFramework\Common\basepage.py", line 46, in get_ele
    ele = self.driver.find_element(*loc)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\webdriver.py", line 282, in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 29, in check_response
    raise wde
  File "D:\Program Files (x86)\python\lib\site-packages\appium\webdriver\errorhandler.py", line 24, in check_response
    super(MobileErrorHandler, self).check_response(response)
  File "D:\Program Files (x86)\python\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: An element could not be located on the page using the given search parameters.

 Wed, 04 Mar 2020 13:16:39  INFO test_0_login.py test_login_tele_wrong [ line:35 ] Test Fail
 Wed, 04 Mar 2020 13:16:40  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Wed, 04 Mar 2020 13:16:40  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
 Wed, 04 Mar 2020 13:17:28  INFO test_0_login.py test_login_tele_wrong [ line:27 ] 手机号输入错误用例
 Wed, 04 Mar 2020 13:17:28  INFO bottom_widget.py switch_bottom_widget [ line:15 ] 切换到页面我
 Wed, 04 Mar 2020 13:17:29  INFO basepage.py visiable_ele [ line:38 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功，耗时0.5458276271820068s
 Wed, 04 Mar 2020 13:17:29  INFO basepage.py get_ele [ line:47 ] 元素('-android uiautomator', 'new UiSelector().text("我")')查找成功！
 Wed, 04 Mar 2020 13:17:29  INFO basepage.py click_ele [ line:80 ] 元素('-android uiautomator', 'new UiSelector().text("我")')成功点击!
 Wed, 04 Mar 2020 13:17:31  INFO basepage.py visiable_ele [ line:38 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功，耗时1.1020066738128662s
 Wed, 04 Mar 2020 13:17:31  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')查找成功！
 Wed, 04 Mar 2020 13:17:32  INFO basepage.py input_ele_text [ line:59 ] 元素('id', 'com.xxzb.fenwoo:id/et_phone')成功输入99999999999
 Wed, 04 Mar 2020 13:17:32  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')查找成功！
 Wed, 04 Mar 2020 13:17:33  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_next_step')成功点击!
 Wed, 04 Mar 2020 13:17:34  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/tv_message')查找成功！
 Wed, 04 Mar 2020 13:17:35  INFO basepage.py get_ele_attr [ line:91 ] 获取元素('id', 'com.xxzb.fenwoo:id/tv_message')得text属性为无效的手机号
 Wed, 04 Mar 2020 13:17:35  INFO test_0_login.py test_login_tele_wrong [ line:33 ] Test Pass!
 Wed, 04 Mar 2020 13:17:35  INFO basepage.py get_ele [ line:47 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')查找成功！
 Wed, 04 Mar 2020 13:17:35  INFO basepage.py click_ele [ line:80 ] 元素('id', 'com.xxzb.fenwoo:id/btn_confirm')成功点击!
